#! /bin/sh /usr/share/dpatch/dpatch-run
## 03-distutils-setup.dpatch by  <alf@merlin>
##
## DP: Use stdlib's distutils for setup. The package list may be generated by:
## DP:  find -name __init__.py | xargs -L1 dirname | grep -v "/testing" | sort

@DPATCH@

--- pylib-0.9.0-beta.orig/setup.py      2007-01-31 23:23:24.000000000 +0100
+++ codespeak-lib-0.9.0-beta/setup.py   2007-02-13 16:32:19.000000000 +0100
@@ -1,3 +1,49 @@
-import py
-from py.__.misc._dist import setup
-setup(py) 
+from distutils.core import setup
+packages=['py',
+          'py.apigen',
+          'py.apigen.rest',
+          'py.apigen.source',
+          'py.apigen.tracer',
+          'py.builtin',
+          'py.code',
+          'py.compat',
+          'py.doc',
+          'py.execnet',
+          'py.io',
+          'py.log',
+          'py.magic',
+          'py.misc',
+          'py.misc.cmdline',
+          'py.path',
+          #'py.path.extpy',
+          'py.path.gateway',
+          'py.path.local',
+          'py.path.svn',
+          'py.process',
+          'py.rest',
+          'py.test',
+          'py.test.rsession',
+          'py.test.rsession.webdata',
+          'py.test.terminal',
+          'py.test.web',
+          'py.thread',
+          'py.tool',
+          'py.xmlobj'
+          ]
+
+scripts=['py/bin/py.cleanup',
+         'py/bin/py.countloc',
+         'py/bin/py.lookup',
+         'py/bin/py.rest',
+         'py/bin/py.test']
+
+package_data={'py.rsession.webdata': ['*.html', '*.js'],
+              'py.apigen': ['*.css', '*.js'],
+              'py.rest': ['*.template'],
+              'py.execnet': ['script/*.py'],
+              }
+
+setup(packages = packages,
+      scripts = scripts,
+      package_data = package_data
+      )
